### DISCUSSION TITLE
3.98.1

### DISCUSSION THREAD

**Discussion by aviramha** [metalbear-co/mirrord#2392] [Announcements]
### Changed

- Internal proxy now emits plain text instead of ANSI

### Fixed

- don't re-resolve when connecting to loopback on outgoing filter
  [#2389](https://github.com/metalbear-co/mirrord/issues/2389)
- Added `JetBrains.Debugger.Worker` to the list of known build tools, fixing
  compatibility with Rider 2024.1.

<hr /><em>This discussion was created from the release <a href='https://github.com/metalbear-co/mirrord/releases/tag/3.98.1'>3.98.1</a>.</em>

**Comment by Amanibhavam** [comment]
@aviramha noticed binaries are missing from this release

**Reply by aviramha** [reply]
we first create a release then CI builds the artifacts and attaches - usually takes 40min
It should be available now.
